bitrise.yml

```diff
@@ -225,6 +225,7 @@ workflows:
         - scheme: FinancialConnections Example
     - slack@3:
         is_always_run: true
+        run_if: .IsBuildFailed
```

Comment:
Docs:
- https://devcenter.bitrise.io/en/steps-and-workflows/introduction-to-steps/enabling-or-disabling-a-step-conditionally.html

See "Running a Step only if the build failed" section